lib/winapi/winbase.h

```diff
@@ -34,6 +34,17 @@ typedef VOID (WINAPI *PFLS_CALLBACK_FUNCTION)(PVOID);
 #define TLS_OUT_OF_INDEXES FLS_OUT_OF_INDEXES
 #define TLS_MINIMUM_AVAILABLE FLS_MAXIMUM_AVAILABLE
 
+typedef struct _INIT_ONCE
+{
+    PVOID Ptr;
+} INIT_ONCE, *PINIT_ONCE, *LPINIT_ONCE;
+#define INIT_ONCE_STATIC_INIT {0}
+#define INIT_ONCE_CTX_RESERVED_BITS 2
+#define INIT_ONCE_CHECK_ONLY 0x00000001UL
+#define INIT_ONCE_ASYNC 0x00000002UL
+#define INIT_ONCE_INIT_FAILED 0x00000004UL
+typedef BOOL CALLBACK PINIT_ONCE_FN(PINIT_ONCE, PVOID, PVOID *);
```

Comment:
nit: Wouldn't one normally write this as `typdef BOOL(CALLBACK *PINIT_ONCE_FN)(PINIT_ONCE, PVOID, PVOID *);`? I think that's how we do it for the kernel functions.